website/docs/cloud-docs/api-docs/workspaces.mdx

```diff
@@ -151,6 +152,31 @@ A run will be triggered in this workspace when changes are detected in any of th
 }
 ```
 
+A run will be triggered in this workspace when changes are detected:
+- in any folder tree which has the leaf folder named `networking` and the change was made in a file with the extension `tf`
+- in any file changed in the folder `/base`, no subfolders are included
+- in any file changed in the folder `/submodule` and all its subfolders
+
+```json
+{
+  "data": {
+    "attributes": {
+      "name": "workspace-4",
+      "resource-count": 0,
+      "terraform_version": "1.2.2",
+      "trigger-patterns": ["/**/networking/*.tf", "/base/*", "/submodule/**/*"],
+      "vcs-repo": {
+        "identifier": "skierkowski/terraform-test-proj-monorepo",
```

Comment:
I think we should change this to something without a name in it :) It's better to be generic in the docs and not include personal identifiers!

```suggestion
        "identifier": example/terraform-test-proj-monorepo",
```

Reply:
agree, should be changed for all 16 occurrences then.